src/VisualStudio/CSharp/Impl/CSharpVSResources.resx

```diff
@@ -540,6 +540,57 @@
   <data name="Event_Hookup_Command_Handler" xml:space="preserve">
     <value>Generate Event Subscription Command Handler</value>
   </data>
+  <data name="Remove_unnecessary_usings" xml:space="preserve">
+    <value>Remove unnecessary usings</value>
+  </data>
+  <data name="Sort_usings" xml:space="preserve">
+    <value>Sort usings</value>
+  </data>
+  <data name="Format_document_settings" xml:space="preserve">
+    <value>Format Document Settings (Experiment) </value>
+  </data>
+  <data name="All_csharp_formatting_rules" xml:space="preserve">
+    <value>All C# Formatting Rules (indentation, wrapping and spacing)</value>
+  </data>
+  <data name="Fix_accessibility_modifiers" xml:space="preserve">
+    <value>Fix accessibility modifiers</value>
+  </data>
+  <data name="Fix_add_remove_braces" xml:space="preserve">
+    <value>Fix add/remove braces</value>
+  </data>
+  <data name="Fix_framework_types" xml:space="preserve">
+    <value>Fix framework types (int vs Int32)</value>
+  </data>
+  <data name="Fix_implicit_explicit_type" xml:space="preserve">
+    <value>Fix implicit/explicit type</value>
+  </data>
+  <data name="Fix_this_qualification" xml:space="preserve">
+    <value>Fix this. qualification</value>
+  </data>
+  <data name="Sort_accessibility_modifiers" xml:space="preserve">
+    <value>Sort accessibility modifiers</value>
+  </data>
+  <data name="Fix_expression_bodied_members" xml:space="preserve">
+    <value>Fix expression-bodied members</value>
+  </data>
+  <data name="Fix_inline_variable_declarations" xml:space="preserve">
+    <value>Fix inline variable declarations</value>
```

Comment:
"Fix" really doesn't make sense here.  It feels better to me as someting that indicates that we're enforcing a style.